api/test/import-data/xlsx-uploads/sourcing-records-import.spec.ts

```diff
@@ -170,4 +170,20 @@ describe('Sourcing Records import', () => {
     const sourcingRecords: SourcingRecord[] = await sourcingRecordRepository.find();
     expect(sourcingRecords.length).toEqual(825);
   });
+
+  test('When a file is sent to the API and gets processed, then a request to Sourcing-Records should return a existing Sourcing-Location ID', async () => {
+    await request(app.getHttpServer())
+      .post('/api/v1/import/sourcing-records')
+      .attach('file', __dirname + '/base-dataset.xlsx');
+
+    const sourcingRecords: SourcingRecord[] = await sourcingRecordRepository.find();
+    const sourcingLocation:
+      | SourcingLocation
+      | undefined = await sourcingLocationRepository.findOne(
+      sourcingRecords[0].sourcingLocationId,
+    );
+
+    expect(sourcingRecords[0].sourcingLocationId).toBeTruthy();
+    expect(sourcingLocation).toBeDefined();
```

Comment:
also do proper validation here, to ensure you are actually getting something that is a sourcingLocation.